### PR TITLE
fix(cves): upgrade pillow version

### LINKED ISSUE #317: fix pillow CVE

### PR DESCRIPTION
**Pull request by jumana-s**
# Description

**What your PR adds/fixes/removes**

Closes #317 

# Tests / Quality Checks

## Automated Testing/build and deployment
- [ ] Does the image pass CI successfully (build, pass vulnerability scan, and pass automated test suite)?
- [ ] If new features are added (new image, new binary, etc), have new automated tests been added to cover these?
- [ ] If new features are added that require in-cluster testing (e.g. a new feature that needs to interact with kubernetes), have you added the `auto-deploy` tag to the PR before pushing in order to build and push the image to ACR so you can test it in cluster as a custom image?

## JupyterLab extensions

- [ ] Are all extensions "enabled" (`jupyter labextension list` from inside the notebook)?

## VS Code tests

- [ ] Does VS Code open?
- [ ] Can you install extensions?

## Code review

- [ ] Have you added the `auto-deploy` tag to your PR before your most recent push to this repo?  This causes CI to build the image and push to our ACR, letting reviewers access the built image without having to create it themselves
- [ ] Have you chosen a reviewer, attached them as a reviewer to this PR, and messaged them with the SHA-pinned image name for the final image to test (e.g. `k8scc01covidacr.azurecr.io/machine-learning-notebook-cpu:746d058e2f37e004da5ca483d121bfb9e0545f2b`)?

### CODE REVIEW

File: docker-bits/2_pytorch.Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+     'pytorch==1.6.0' \
+     'torchvision==0.7.0' \
+     'ipykernel==5.3.4' \
+   && \
+   conda install -n torch --quiet --yes -c pytorch \
+     'torchtext==0.7.0' \
+   && \
+   conda clean --all -f -y && \
+   fix-permissions $CONDA_DIR && \
+   fix-permissions /home/$NB_USER
```
Comment (bryanpaget) [inline docker-bits/2_pytorch.Dockerfile:13]: just whitespace cleanup

**Review by bryanpaget — APPROVED** [review]
Code is a bit repetitive between Dockerfiles. Should probably consider a common Dockerfile for the overlapping packages.

**Comment by jumana-s** [comment]
Yeah, we are due for cleaning up the docker bits.

File: docker-bits/2_pytorch.Dockerfile (hunk 1 above)
Comment (jumana-s) [reply docker-bits/2_pytorch.Dockerfile:13]: should I get rid of it?

**Review by bryanpaget — COMMENTED** [review]
Pillow has been updated to 9.0.0.

**Review by bryanpaget — COMMENTED** [review]
Looks good to me. :-) Updated to Pillow 9.0.0 except for the PyTorch image. Fixing PyTorch image depends on https://github.com/conda-forge/pillow-feedstock/pull/104

**Review by bryanpaget — APPROVED** [review]

**Pull request merged**